mods/tnt/init.lua

```diff
@@ -1,11 +1,6 @@
 tnt = {}
--- Default to enabled in singleplayer and disabled in multiplayer
-local singleplayer = minetest.is_singleplayer()
-local setting = minetest.setting_getbool("enable_tnt")
-if (not singleplayer and setting ~= true) or
-		(singleplayer and setting == false) then
-	return
-end
+-- Default to enabled
+local enable_tnt = minetest.setting_getbool("enable_tnt") ~= false
```

Comment:
This is not equivalent behaviour. Currently it disables tnt if:
multiplayer and setting is nil or false (~= true)
singleplayer and setting is false
This is to be more cautious in multiplayer.